nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -483,6 +510,7 @@ After going through this checklist, you're ready to go!
 	obj_integrity = max_integrity
 	RegisterSignal(src, COMSIG_MOVABLE_MOVED, .proc/check_overmap_elegibility) //Used to smoothly transition from ship to overmap
 	RegisterSignal(src, COMSIG_AREA_ENTERED, .proc/update_overmap) //Used to smoothly transition from ship to overmap
+	add_overlay(image(icon = icon, icon_state = "canopy_open", dir = SOUTH))
 
 /obj/structure/overmap/fighter/proc/prebuilt_setup()

```

Comment:
So do escape pods still have 6 torpedoes and just can't use them?

Reply:
God that's awful. I love it